main/main.go

```diff
@@ -469,6 +455,44 @@ func createApplicationRelayersForSourceChain(
 	return applicationRelayers, minHeight, nil
 }
 
+func createDeciderClient(host string, port *uint16) (*grpc.ClientConn, error) {
+	if port == nil {
+		return nil, nil
+	}
+
+	if len(host) == 0 {
+		host = "localhost"
+	}
+
+	uri := strings.Join(
+		[]string{host, strconv.FormatUint(uint64(*port), 10)},
+		":",
+	)
+
+	_, err := url.ParseRequestURI(uri)
+	if err != nil {
+		return nil, fmt.Errorf("Invalid URI: %w", err)
+	}
```

Comment:
I know that functionally this results in the same outcome (the application exiting), but can you move or replicate this config validation code to live in `package config` for consistency?

Reply:
ah I forgot about that existing validation logic, thanks for pointing this out. addressed in https://github.com/ava-labs/awm-relayer/pull/344/commits/a9c374c2377ed4b33ad86dafab7610c6881ffa40